src/Microsoft.SqlTools.ManagedBatchParser/BatchParser/BatchParserWrapper.cs

```diff
@@ -51,7 +51,7 @@ private static List<BatchDefinition> ConvertToBatchDefinitionList(string content
                     int offset = offsets[0];
                     int startColumn = batchInfos[0].startColumn;
                     int count = batchInfos.Count;
-                    string batchText = content.Substring(offset, batchInfos[0].length);
+                    string batchText = batchInfos[0].batchText; // content.Substring(offset, batchInfos[0].length);
```

Comment:
>// content.Substring(offset, batchInfos[0].length); [](start = 64, length = 51)

This seems fine (was it broken before) but get rid of the commented out code

Reply:
This was broken in context of substitute text which was not used from anywhere. will remove the comment, thanks for pointing out